.github/workflows/release-nuget.yaml

```diff
@@ -12,10 +12,12 @@ jobs:
     environment: Release
     steps:
       - uses: actions/checkout@v4
+      
       - name: Setup .NET
         uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: 8.0.x
```

Comment:
@gasparnagy - are you OK with changing the framework version(s) for cucumber/messages? I think we're the only  consumers at this point. 
If I understand correctly, and please confirm, our policy is that anything published as a nuget package (such as Cucumber.Messages) should target .NET Standard 2.0 (only) and that these can be built and tested on .net 8?

Reply:
this line change only sdk version for release github action.
.NET standard 2.0 is still in tfm list

Reply:
Yes, this is fine and as @Romfos mentioned this is just what SDK we use to build. As long as we target .NET Standard 2.0 it should be all good.